arches/app/models/models.py

```diff
@@ -1138,7 +1184,7 @@ class TileModel(models.Model):  # Tile
     resourceinstance = models.ForeignKey(ResourceInstance, db_column="resourceinstanceid", on_delete=models.CASCADE)
     parenttile = models.ForeignKey("self", db_column="parenttileid", blank=True, null=True, on_delete=models.CASCADE)
     data = JSONField(blank=True, null=True, db_column="tiledata")
-    nodegroup = models.ForeignKey(NodeGroup, db_column="nodegroupid", on_delete=models.CASCADE)
+    nodegroup_id = models.UUIDField(db_column="nodegroupid", null=True)
```

Comment:
Heads up that the first failing test on the last push looks like it was due to this change, so let's add a breaking changes notice. There could be applications that need to check their views & tests.